src/app/locator/qgsinbuiltlocatorfilters.cpp

```diff
@@ -400,6 +401,8 @@ void QgsAllLayersFeaturesLocatorFilter::fetchResults( const QString &string, con
       result.userData = QVariantList() << f.id() << preparedLayer.layerId;
       result.icon = preparedLayer.layerIcon;
       result.score = static_cast< double >( string.length() ) / result.displayString.size();
+
+      result.contextMenuActions.insert( OpenForm, new QAction( tr( "Open form" ) ) );
```

Comment:
@nyalldawson I guess there is a leak there. I am not sure how to handle this. Is it the responsibility of the model to delete the actions when it's cleared?